index.html

```diff
@@ -2,265 +2,400 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Mens Portfolio</title>
 
-    <link rel="stylesheet" href="css/index.css" />
-    <link rel="stylesheet" href="css/works.css" />
-    <link rel="stylesheet" href="css/about.css" />
-    <link rel="stylesheet" href="css/contact.css" />
-    <link rel="stylesheet" href="css/desktop.css" />
+    <link rel="stylesheet" href="css/index.css">
+    <link rel="stylesheet" href="css/works.css">
+    <link rel="stylesheet" href="css/about.css">
+    <link rel="stylesheet" href="css/contact.css">
+    <link rel="stylesheet" href="css/desktop.css">
+    <link rel="stylesheet" href="css/pop_up_mobile.css">
+    <link rel="stylesheet" href="css/pop_up_dsktop.css">
+
+  
+      
 </head>
 
 <body>
+
+    <div id="projects_modal">
+        <div class="project_modal_grid">
+            <div class="project_grid_row">
+                <span class="close2">&times;</span>
+            </div>
+            <div class="project_grid_row">
+                <h1 class="project_header text_color text_body_font m_header">Project name goes here</h1>
+            </div>
+            <div class="project_grid_row grid_img_2">
+                <ul class="project_category list_category text_body_font">
+                    <li>Html</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+            </div>

```

Comment:
- [ ] Good work creating the projects array! However, as the previous reviewer said, the goal here is to load the projects' data from the `projectlist1` array, right now the info displayed on the modal window is hardcoded:

Data displayed:
![Captura de pantalla 2022-04-16 a la(s) 3 44 11 p  m](https://user-images.githubusercontent.com/83717285/163690807-459d41dc-a3e1-4292-af95-bd3fd8f92f26.png)

Data that should be displayed:
![Captura de pantalla 2022-04-16 a la(s) 3 39 01 p  m](https://user-images.githubusercontent.com/83717285/163690826-bd37a63b-ab70-4eb4-8756-3feb0b31281b.png)

- Please remove the hardcoded data as the previous CR said and populate it from the array, this way your site will be more dynamic.